tests/playwright/pages/editor-page.ts

```diff
@@ -509,13 +509,13 @@ export default class EditorPage extends BasePage {
 	async publishPage() {
 		await this.page.locator( 'button#elementor-panel-saver-button-publish' ).click();
 		await this.page.waitForLoadState();
-		await this.page.getByRole( 'button', { name: 'Update' } ).waitFor();
+		await this.page.locator( '#elementor-panel-saver-button-publish-label' ).waitFor();
 	}
 
 	async publishAndViewPage() {
 		await this.publishPage();
 		await this.page.locator( '#elementor-panel-header-menu-button i' ).click();
-		await this.page.getByRole( 'link', { name: 'View Page' } ).click();
```

Comment:
What is the best way how to deal with text selector when they can be translated into another language (Hebrew in this case)?

Reply:
@dmaziar For the moment I don't strictly need the translations. I have selected Hebrew as a language, so that I can test the RTL view, but the Hebrew translation isn't installed on our WordPress setup, so the text will still be displayed in English. In this way, we can test RTL, without having the language problem.

Once I download the translations from WordPress, then everything starts to break. I have removed that step for now.

I hope I am making sense.

But in general, I would like to know what would be the best approach here. It might be relevant in the future.

Reply:
I think nothing we can do here, except using ids and classes. I'm pretty sure Playwright is not the right tool for localisation testing. We will have to think about the best solution.